TeaDiary/Scripts/App/Teas/TeasCreatePage.js

```diff
@@ -0,0 +1,18 @@
+﻿function onCreateClick() {
```

Comment:
You don't need to add `page` postfix to a script file. If there is `Teas/xxx.cshtml`, I'd be fine with just `Teas/xxx.js`